package.json

```diff
@@ -13,7 +13,8 @@
     "build": "tsc",
     "clean": "rimraf build/*",
     "cleanTests": "rimraf build/**/*.test.*",
-    "lint": "eslint */**.ts",
+    "lint": "eslint source/**.ts",
+    "fix": "eslint --fix source/**.ts",
```

Comment:
Lint kept checking the build directory and informing that it skipped checking the `.d.ts` files, but I assumed you don't really want to lint your output.

I added auto-fix to eslint, so it would help me with the demanded line endings.